qemu/tests/cfg/nested_hypev_on_kvm.cfg

```diff
@@ -0,0 +1,18 @@
+- nested_hyperv_on_kvm:
+    type = nested_hyperv_on_kvm
```

Comment:
Your qemu/tests/nested_hypev_on_kvm.py shoud change to qemu/tests/nested_hyperv_on_kvm.py, otherwise, avocado will report "ERROR: Could not find test file nested_hyperv_on_kvm.py on testdirs"

Reply:
Thank you so much for review, done.